### DISCUSSION TITLE
2021.02.15 월 회의록

### DISCUSSION THREAD

**Discussion by hakju** [hakju/ios-cloud-notes#2] [General]
### [ 기능 요구서 ]
  1. TableView 코드로 구현
  2. 오토레이아웃 (리스트의 각 행에는 제목, 작성일자, 한줄 미리보기 정보가 표시)
  3. 작성일자는 사용자의 지역 포멧에 맞게 표현
  4. JSON 데이터를 활용, 타입 구현
  5. 수정, 삭제, 자동탐지(?)
  6. Size class
  - Step 1 PR은 늦어도 금요일에 보내기
  
  ### [ 계획 ]
  2021.02.15 월
  1. TableView 코드로 구현해보기
  2. TIL 작성하기

**Comment by soleJin** [comment]
+ TableView 코드로 구현
+ 오토레이아웃 공부
